### ISSUE TITLE
[Win32][Accessibility] [A11yMAS][Keyboard Navigation] [MAS2.1.1][Run Task pane is not accessible by using keyboard.]

### ISSUE THREAD

**Issue by DasaradhaKamjula** [OfficeDev/script-lab#732]
**User Experience:**
User who rely on keyboard will not be able to access those controls if they are not accessible by keyboard which may lead for external help.

**Test Environment:**
OS Build: 16299.19
Power point version:1712(8808.1000)
**Prerequisites:**
Add script lab “Add ON” In Power point

**Attachments:**
[MAS2.1.1_Run task pane.zip](https://github.com/OfficeDev/script-lab/files/1478219/MAS2.1.1_Run.task.pane.zip)

**Repro Steps:**
1.Open Script Lab in Power point.
2.Navigate to Run button and activate it.
3.Try to Navigate to Run pane Using F6 key.
4.Verify whether user is able to access Run task pane using Keyboard.

**Current Behavior:**
“Run task pane" is not accessible by Keyboard.

**Expected Behavior:**
“Run task pane” should be accessible by Keyboard. So that keyboard user can be able to access all controls without any problem.

**MAS Reference:**
https://microsoft.sharepoint.com/%3Aw%3A/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc=%7B8492c4eb-c179-40ae-8777-cd044ed725a2%7D

**Suggested Fix:**
https://microsoft.sharepoint.com/teams/msenable/mas/pages/mas-details.aspx?masid=2.1.1
